src/psyclone/psyir/nodes/structure_member.py

```diff
@@ -101,6 +102,22 @@ def _validate_child(position, child):
         # Only one child is permitted
         return False
 
+    @property
+    def member(self):
+        '''
+        :returns: the member of the structure that is being accessed.
+        :rtype: (sub-class of) :py:class:`psyclone.psyir.nodes.Member`
+
+        :raises InternalError: if the first child of this node is not an \
+                               instance of Member.
+        '''
+        if not isinstance(self.children[0], Member):
+            raise InternalError(
+                "StructureMember malformed or incomplete. The first child "
```

Comment:
Since this can be inherited by AoSMember, can you replace "StructureMember" to `type(self).__name__` or something similar.

Reply:
Oh yes, done.